test/exercises/acronym_test.rb

```diff
@@ -1,7 +1,11 @@
 require "test_helper"
 require 'pry'
```

Comment:
Is this used in this source?

If using Pry externally, I suggest not making it a dependency (I have removed this dependency in #119 ) and using `ruby -rpry` or otherwise requiring it on demand when it is needed.